packages/plugin-ext/src/plugin/documents.ts

```diff
@@ -221,6 +221,7 @@ export class DocumentsExtImpl implements DocumentsExt {
             // return opened document
             return document;
         } catch (error) {
+            console.log(error);
```

Comment:
left overs? Should not it be handled by clients since we throw it?

Reply:
done, fixed